app/src/main/java/com/rlab/sejima/features/MUHeader.java

```diff
@@ -88,51 +103,158 @@ public MUHeader(Context context, AttributeSet attrs) {
         CharSequence s;
 
         // Deal with title's attributes
-        s = a.getString(R.styleable.MUHeader_title);
+        s = a.getString(R.styleable.MUHeader_header_title);
         mTitle = TextUtils.isEmpty(s) ? mTitle : s.toString();
-
-        mTitleColor = a.getColor(R.styleable.MUHeader_title_color, mTitleColor);
-        mTitleSize = a.getDimensionPixelSize(R.styleable.MUHeader_title_size, 0);
-        mTitleWeight = a.getInt(R.styleable.MUHeader_title_weight, mTitleWeight);
+        mTitleColor = a.getColor(R.styleable.MUHeader_header_title_color, mTitleColor);
+        mTitleSize = a.getDimensionPixelSize(R.styleable.MUHeader_header_title_size, 0);
+        mTitleWeight = a.getInt(R.styleable.MUHeader_header_title_weight, mTitleWeight);
 
         // Deal with detail's attributes
-        s = a.getString(R.styleable.MUHeader_detail);
+        s = a.getString(R.styleable.MUHeader_header_detail);
         mDetail = TextUtils.isEmpty(s) ? mDetail : s.toString();
+        mDetailColor = a.getColor(R.styleable.MUHeader_header_detail_color, mTitleColor);
+        mDetailSize = a.getDimensionPixelSize(R.styleable.MUHeader_header_detail_size, 0);
+        mDetailWeight = a.getInt(R.styleable.MUHeader_header_detail_weight, mDetailWeight);
 
-        mDetailColor = a.getColor(R.styleable.MUHeader_detail_color, mTitleColor);
-        mDetailSize = a.getDimensionPixelSize(R.styleable.MUHeader_detail_size, 0);
-        mDetailWeight = a.getInt(R.styleable.MUHeader_detail_weight, mDetailWeight);
-
-        mVerticalSpacing = a.getDimensionPixelSize(R.styleable.MUHeader_vertical_spacing, mVerticalSpacing);
-        mTextAlignment = a.getInt(R.styleable.MUHeader_text_alignment, ALIGN_START);
+        mVerticalSpacing = a.getDimensionPixelSize(R.styleable.MUHeader_header_vertical_spacing, 0);
+        mAlignment = a.getInt(R.styleable.MUHeader_header_alignment, mAlignment);
 
         init(context);
         a.recycle();
     }
 
     private void init(Context context) {
-        inflate(context, R.layout.feature_mu_header, this);
+        float scale = (float) context.getResources().getDisplayMetrics().densityDpi / DisplayMetrics.DENSITY_DEFAULT;
```

Comment:
To avoid the calcul on Title/Device size and spacing, can we store this variable as a private one? It's just one division so not an heavy calcul.

Reply:
sure.